src/Microsoft.ML.Data/Prediction/Calibrator.cs

```diff
@@ -981,12 +989,12 @@ private static VersionInfo GetVersionInfo()
         }
 
         private readonly IHost _host;
-        private readonly Float _binSize;
-        private readonly Float _min;
-        private readonly Float[] _binProbs;
+        private readonly float _binSize;
+        private readonly float _min;
+        private readonly float[] _binProbs;
```

Comment:
make them public, make _binProbs ImmutableArray.

Reply:
Are you make them public in #1871 or you don't think they should be public?

---
In reply to: [241147679](https://github.com/dotnet/machinelearning/pull/1816#discussion_r241147679) [](ancestors = 241147679)